include/deal.II/lac/tridiagonal_matrix.h

```diff
@@ -241,10 +241,10 @@ class TridiagonalMatrix
   /**
    * Output of the matrix in user-defined format.
    */
-  template <class OUT>
-  void print (OUT &s,
-              const unsigned int  width=5,
-              const unsigned int  precision=2) const;
+  template <class OUT_>
+  void print(OUT_ &s,
```

Comment:
`OUT` is supposed to be a stream type. Can you rename the template type to `OutputStream` instead?